services/account_type/legacy.yaml

```diff
@@ -132,13 +133,6 @@ Resources:
                 - !Sub 'arn:aws:s3:::${AuditCloudTrailBucketName}/*'
             - !Ref AWS::NoValue
           # Master Payer Account Permissions
-          - !If
```

Comment:
Based on our conversations @silvexis, new plan is to simply unconditionally apply these "master payer" permissions but _keep_ the conditional application of the bucket permissions. We will still be able to debug b/c we have `s3:List*` below